src/server/index.js

```diff
@@ -1,11 +1,21 @@
 const express = require('express');
-
+// react runs on port 3000 by default
+const port = 9000;
 const app = express();
 
+// for this to work you might need to run 
+// $npm install --save cors
```

Comment:
As long as you do `npm install` before launching, any dependencies you're missing but are in the package.json should be automatically installed, so I don't think this comment is necessary

Reply:
Sounds good, will remove it